quicksort/quicksort.py

```diff
@@ -0,0 +1,16 @@
+def s(l,st,end):
+    print("st,end: ", st, end)
+    if len(l) <= 1: return
```

Comment:
this base case is incorrect:

The length of l never changes in recursive calls so this will never trigger and you will go into an infinite loop

since you are not checking for the correct base case, your code is getting into an invalid state which is why it is crashing.

Reply:
That was the reason - fixed it - worked.